node-commons/src/main/java/org/apache/iotdb/commons/partition/DataPartitionQueryParam.java

```diff
@@ -28,12 +28,31 @@ public class DataPartitionQueryParam {
   private String devicePath;
   private List<TTimePartitionSlot> timePartitionSlotList = new ArrayList<>();
 
+  // it will be set to true in query when there exist filter like: time <= XXX
+  // (-oo, timePartitionSlotList.get(0)]
+  private boolean needLeftAll = false;
+
+  // it will be set to true query when there exist filter like: time >= XXX
+  // [timePartitionSlotList.get(timePartitionSlotList.size() - 1), +oo)
+  private boolean needRightAll = false;
```

Comment:
The annotation shows that these two parameters indicate two close intervals. However in the implementation of SeriesPartitionTable, these two parameters indicate two open intervals.

Reply:
The margin partition will be added （if need）in  previous process code. 
![image](https://user-images.githubusercontent.com/60659567/204289763-edd6d629-5bcc-47b9-8a2d-068d337814a6.png)